src/coreclr/vm/peassembly.h

```diff
@@ -109,54 +111,15 @@ class PEFile
     CHECK Invariant();
 #endif
 
-private:
-    // ------------------------------------------------------------
-    // Loader access API
-    // ------------------------------------------------------------
-
-    friend class DomainFile;
-
-public:
-    void LoadLibrary(BOOL allowNativeSkip = TRUE);
-
-
-private:
-    // For use inside LoadLibrary callback
-    void SetLoadedHMODULE(HMODULE hMod);
-
-    // DO NOT USE !!! this is to be removed when we move to new fusion binding API
-    friend class DomainAssembly;
```

Comment:
None of these really needs to be a friend. `Assembly` would need to be a friend because of `DefineEmitScope`, but `DefineEmitScope` is itself just a helper that does not use anything in `PEAssembly` and can be moved to where it is used.